### DISCUSSION TITLE
Insecure Webview component for 3rd party integration use case

### DISCUSSION THREAD

**Discussion by AppSeginf** [OWASP/mastg#2425] [Ideas]
As you may all know, using webviews to render 3rd party flows, like logins, impose a risk for the end-user information because of the fact the webviews run in the application user space, among other security warnings. 

This is why Google (https://developer.chrome.com/docs/android/custom-tabs/#when-should-i-use-custom-tabs-vs-webview) and other broad list of companies (like Amazon https://developer.amazon.com/es/docs/login-with-amazon/webview.html) don't recommend using webview to load 3rd party arbitrary code.

**Comment by cpholguera** [comment]
Absolutely! If not already there we must add this to the new MASTG tests. Have you seen our latest news?

https://mas.owasp.org/news/

Here's the proposal for new MASTG tests, feel free to add all your suggestions for changes or additions of new tests:
 https://docs.google.com/spreadsheets/d/1Go5GpVvKJqTDxGbSLBPZb1hmYi5lXRc1D1AfrTTkUkY/edit?usp=sharing

Thanks a lot @AppSeginf!
